lsp/client/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "lsp-sample-client",
+    "description": "VSCode part of a language server",
+    "author": "Microsoft Corporation",
+    "license": "MIT",
+    "version": "0.0.1",
+    "publisher": "vscode",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/vscode-extension-samples"
+    },
+    "engines": {
+        "vscode": "1.77.0"
+    },
+    "dependencies": {
+        "vscode-languageclient": "^9.0.1"
+    },
+    "devDependencies": {
+        "@types/vscode": "1.77.0",
+        "@vscode/test-electron": "^2.3.9"
+    }
+}
```

Comment:
Can we update package.json for client and server with appropriate values.

Reply:
Change it to `mobile-lsp-server` and `mobile-lsp-client`